example/monteCarloIntegration/src/monteCarloIntegration.cpp

```diff
@@ -105,12 +105,11 @@ auto main() -> int
 
     // Setup buffer.
     BufHost bufHost{alpaka::allocBuf<uint32_t, Idx>(devHost, extent)};
-    uint32_t* const ptrBufHost{alpaka::getPtrNative(bufHost)};
     BufAcc bufAcc{alpaka::allocBuf<uint32_t, Idx>(devAcc, extent)};
     uint32_t* const ptrBufAcc{alpaka::getPtrNative(bufAcc)};
```

Comment:
if you like it better, you could also use
```suggestion
    uint32_t* const ptrBufAcc = std::data(bufAcc);
```
or
```suggestion
    uint32_t* const ptrBufAcc = bufAcc.data();
```

Reply:
I like this, it looks more like STL and explicitly says that we like to use the buffer as contiguous array.

Reply:
Thanks, great. I replaced all `alpaka::getPtrNative `calls with` std::data `in all examples and tests under `test/integ`. ( in 2 different commits. )